src/routes/api/users/UserController/UserController.js

```diff
@@ -0,0 +1,30 @@
+import db from '../../../../models/index';
+/**
+ *
+ *
+ * @class Users
+ */
+class Users {
+  /**
+   *
+   *
+   * @static
+   * @param {obj} req
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const result = await db.user
```

Comment:
result? You could have simply used users

Reply:
Thanks for the feedback. I have implemented it.